website/src/content/deployment/serving-static-assets.mdx

```diff
@@ -31,6 +31,13 @@ For example, Merchant Center static assets are uploaded to a Google Cloud Storag
 }
 ```
 
+To serve static assets on a remote CDN or Cloud Storage, do the following:
```

Comment:
Using the `compile-html` command does not necessarily mean that the assets should be served by a CDN storage.

In fact, if you use services like Vercel, etc, the assets are managed by the hosting service itself. That's why you provide the `use-local-assets` option.

Therefore, we should mention this in this page as well.

It might be that we need to restructure a bit the sections to do so.

Reply:
Do you want to take over this part?